### PR TITLE
Replace fluxible with react-router

### PR DESCRIPTION
**Pull request by DanielHeath**
At this point, the isomorphic stuff works beautifully and the app is drastically simpler.

### CODE REVIEW

File: src/server/render.js [hunk 1]
```diff
@@ -2,80 +2,47 @@
...
+            const html = React.renderToStaticMarkup(
+              <HtmlDocument
+                markup={markup}
+                script={webpackStats.script}
+                css={webpackStats.css}
+              />
+            );
+            const doctype = "<!DOCTYPE html>";
+            res.send(doctype + html);
+        });
```
Comment (danni) [inline src/server/render.js:40]: I don't understand why we're running the router in 2 places.

Comment (DanielHeath) [inline src/server/render.js:40]: Once on the server (when a request comes in) and once on the client (on `window.history.push`)

File: src/routes.js [hunk 2]
```diff
@@ -1,34 +1,26 @@
...
-    },
-
-    // This route doesn't point to any handler.
-    bad: {
-        path: "/bad",
-        method: "get",
-        action: InitActions.badPage,
-    },
-};
+export default <Router.Route
```
Comment (danni) [inline src/routes.js:10]: Syntax error in Travis. Should this be a .jsx file? Do you need to wrap the XML node in brackets?

Comment (DanielHeath) [inline src/routes.js:10]: just looking at this now.

**Pull request merged**